src/services/users/index.js

```diff
@@ -51,6 +73,15 @@ const createNewUser = async (user) => {
 		data: {user : body}, 
 		expiry : '15m'
 	})
+
+	await db.query(
+		`UPDATE user_invites 
+		SET status = $1 
+		WHERE email = $2 
+		RETURNING *`, 
```

Comment:
@itzgeebee we will not need the `RETURNING *` line here because its response is not being assigned to any variable